public/app/features/dashboard-scene/edit-pane/DashboardEditPaneSplitter.tsx

```diff
@@ -70,12 +73,16 @@ export function DashboardEditPaneSplitter({ dashboard, isEditing, body, controls
 
   return (
     <div {...containerProps} style={containerStyle}>
-      <div {...primaryProps} className={cx(primaryProps.className, styles.canvasWithSplitter)}>
+      <div
+        {...primaryProps}
+        className={cx(primaryProps.className, styles.canvasWithSplitter)}
+        onPointerDown={() => editPane.clearSelection()}
```

Comment:
It would be great to have ESC as another way to unselect the selected element. WDYT?